department-of-veterans-affairs/caseflow
# Refactor HearingDayRange

### Description
This is a refactor of the `HearingDayRange`

### Acceptance Criteria
- [ ] The Hearings index page /hearings/schedule should load for all users (VSO, judges and the rest)

### Testing Plan
1. Shadow user BVARERDMAN
2. Go to /hearings/schedule The page should load the HearingDays
3. Shadow user BILLIE_VSO
4. Go to /hearings/schedule The page should load the HearingDays
5. Shadow user BVASYELLOW
6. Go to /hearings/schedule The page should load the HearingDays
